### PR TITLE
tproxy expression support and test

### PR DESCRIPTION
**Pull request by sbezverk**
Closes: #31

### CODE REVIEW

**Comment by sbezverk** [comment]
@stapelberg Would be great if you could review it, thanks a lot.

File: expr/tproxy.go [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+import (
+	"encoding/binary"
+
+	"github.com/google/nftables/binaryutil"
+	"github.com/mdlayher/netlink"
+	"golang.org/x/sys/unix"
+)
+
+// IPv4
```
Comment (stapelberg) [inline expr/tproxy.go]: Can you remove these notes please?

File: nftables_test.go [hunk 2]
```diff
@@ -561,6 +561,70 @@ func TestAddCounter(t *testing.T) {
...
+				if got, want := b, want[0]; !bytes.Equal(got, want) {
+					t.Errorf("message %d: %s", idx, linediff(nfdump(got), nfdump(want)))
+				}
+				want = want[1:]
+			}
+			return req, nil
+		},
+	}
+
+	//	[ payload load 1b @ network header + 9 => reg 1 ]
```
Comment (stapelberg) [inline nftables_test.go]: Can you interleave these with the expressions in line 612 and following please?

Comment (sbezverk) [reply nftables_test.go]: done

File: expr/tproxy.go (hunk 1 above)
Comment (sbezverk) [reply expr/tproxy.go]: done

**Comment by stapelberg** [comment]
Thanks!

**Pull request merged**